services/cooperation.js

```diff
@@ -2,21 +2,22 @@ const Cooperation = require('~/models/cooperation')
 
 const cooperationService = {
   getCooperations: async () => {
-    return await Cooperation.find().lean().exec()
+    return await Cooperation.find().populate('offer', ['id', 'author', 'price']).lean().exec()
   },
 
   getCooperationById: async (id) => {
-    return await Cooperation.findById(id).lean().exec()
+    return await Cooperation.findById(id).populate('offer', ['id', 'author', 'price']).lean().exec()
   },
 
   createCooperation: async (initiatorUserId, data) => {
-    const { offerId, recipientUserId, price } = data
+    const { offer, requiredTutoringLevel, requiredLanguage, additionalInfo } = data
```

Comment:
why you removed the price? look at uml diagram and figma, price can be changed after creating coop and it will be separated from offers price, if you want you can take price from offer by offerId

Reply:
you are right, already changed